api/app/src/Entity/StagingDeputyship.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use League\Csv\Serializer;
+
+/**
+ * Holds staging data for processing deputyship CSV file from Sirius.
+ * Does no validation, has no de-duplication, and has no foreign keys.
+ * It has a unique key, but only because Doctrine demands one.
+ *
+ * Column names in the database (as defined here) *must* match the column names in the CSV, as we do no
+ * transformation/translation when dumping data into this table.
+ *
+ * @ORM\Table(name="deputyship", schema="staging", indexes={
+ *
+ *   @ORM\Index(name="deputy_uid_idx", columns={"DeputyUid"}),
+ *   @ORM\Index(name="case_number_idx", columns={"CaseNumber"}),
```

Comment:
Just deputy_uid_idx + order_uid_idx are needed for uniqueness (the court order, if repeated, would share the same casenumber anyway)